alert-policies/apache-traffic-server/SSLExpiredCertificate.yml

```diff
@@ -0,0 +1,30 @@
+# Name of the alert
+name: SSL Expired Certificates
+
+# Description and details
+description: |+
+  This alert is triggered when the SSL certificate is within 15 days of expiration.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+
+  query: "SELECT latest(global.proxy.process.ssl.origin_server_expired_cert) AS 'ssl expired' FROM ATSSampleMetrics"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 7
```

Comment:
should this be changed ?
as the alert has to be within 15 days.

Reply:
Hi @pkosanam ,
Here changed the entire alert description and threshold. Kindly review it.

Thanks...

Thanks...